ocean_lib/ocean/ocean.py

```diff
@@ -117,13 +119,14 @@ def OCEAN_address(self) -> str:
         return get_ocean_token_address(self.config.address_file, web3=self.web3)
 
     @enforce_types
-    def create_data_token(
+    def create_nft_token(
         self,
         name: str,
         symbol: str,
+        token_uri: str,
         from_wallet: Wallet,
-        cap: int = DataToken.DEFAULT_CAP,
-        blob: str = "",
+        template_index: Optional[int] = 1,
+        additional_erc20_deployer: Optional[str] = None,
     ) -> DataToken:
```

Comment:
This should return ERC721Token. I have an example for this function [here](https://github.com/oceanprotocol/ocean.py/blob/simple-flow/ocean_lib/ocean/ocean.py#L122)

Reply:
you're right, I'll change the return type and handle your docstrings comment too.